index.bs

```diff
@@ -306,6 +306,10 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
     [=credential key pair=] is known as the <dfn>credential private key</dfn>. Note that in the case of [=self
     attestation=], the [=credential key pair=] is also used as the [=attestation key pair=], see [=self attestation=]
     for details.
+ 
+: <dfn>Rate Limiting</dfn>
+:: Rate limiting is the process by which an authenticator limits the number of unsuccessful authentication attemps within a set
```

Comment:
ah, UAF authenticator commands "[security guidelines](https://fidoalliance.org/specs/fido-uaf-v1.1-id-20170202/fido-uaf-authnr-cmds-v1.1-id-20170202.html#security-guidelines)" says in the "Matcher" section:
> Authenticators must implement anti-hammering protections for their matchers.

which it seem my S5 is doing.